modin/pandas/test/test_io.py

```diff
@@ -277,8 +277,8 @@ def test_read_csv_col_handling(
         skip_blank_lines,
     ):
         if request.config.getoption("--simulate-cloud").lower() != "off":
-            pytest.skip(
-                "The reason of tests fail in `cloud` mode is unknown for now - issue #2340"
+            pytest.xfail(
```

Comment:
Is there a reason why this xfail is moved into the method, while for the other files, it is above the def?

Reply:
Sorry I do not understand what you mean.

Reply:
The other tests that are expected to fail have xfail declared above the function - but these functions have xfail inside the codepath - I was wondering if there was a specific reason for the difference.